### PR TITLE
Register Eirene [https://github.com/Eetion/Eirene.jl.git]

### PR DESCRIPTION
**Pull request by Eetion**
…l.git]

### CODE REVIEW

**Comment by tkelman** [comment]
the email in the url should be replaced with a github username. is this a publicly visible repo?

**Comment by Eetion** [comment]
Thank you!  I think the repo is publicly visible now.

File: Eirene/url [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/greg.henselman@gmail.com/Eirene.jl.git
```
Comment (tkelman) [inline Eirene/url]: you'll need to update the url here, and tag a version

**Comment by tkelman** [comment]
See https://docs.julialang.org/en/release-0.5/manual/packages/#generating-the-package, or look at the structure of existing packages, for how a package should be structured. You should declare your dependencies with a REQUIRE file, put your julia code in a file named `src/Eirene.jl` that defines a `module Eirene`, and add some unit tests in a file named `test/runtests.jl`.

**Comment by jingpengwu** [comment]
@Eetion I suggest using attobot, it is really easy to release code!

**Comment by Eetion** [comment]
@tkelman and @jingpengwu, thank you!  I've been a bit swamped so progress is slow, but will follow up with attobot.  best.

**Comment by MikeInnes** [comment]
Closing this as we haven't heard from you in a while. Do let me know if you want to re-open.

**Comment by Eetion** [comment]
Thanks for the message - I've been tied up the past few months but do plan to continue, hopefully starting mid November.

**Pull request closed without merging**